plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -164,6 +169,24 @@ type ExtenderFilterResult struct {
 	Error string `json:"error,omitempty"`
 }
 
+// Binding represents the binding of a pod to a node.
+type Binding struct {
+	// PodName is the name of the pod being bound
+	PodName string
+	// PodNamespace is the namespace of the pod being bound
+	PodNamespace string
+	// PodUID is the UID of the pod being bound
+	PodUID types.UID
```

Comment:
In order to post the binding to apiserver, the Binding object in pkg/api/types.go needs to be filled with pod name, namespace also.